libs/vgc/ui/widget.cpp

```diff
@@ -392,7 +415,7 @@ void Widget::clearFocus()
     Widget* w = focusedWidget();
     if (w) {
         if (isTreeActive()) {
-            onFocusOut();
+            w->onFocusOut();
```

Comment:
Note : This fixes a focus bug: when clicking from a line edit to another, the cursor was still visible in the previously focused line edit.